VitruvioHost/Plugins/Vitruvio/Source/VitruvioEditor/Private/VitruvioStyle.cpp

```diff
@@ -0,0 +1,59 @@
+#include "VitruvioStyle.h"
+
+#include "Styling/SlateStyleRegistry.h"
+#include "Interfaces/IPluginManager.h"
+#include "SlateOptMacros.h"
+
+#define IMAGE_BRUSH_SVG( RelativePath, ... ) FSlateVectorImageBrush( Style->RootToContentDir(RelativePath, TEXT(".svg")), __VA_ARGS__)
+
+TSharedPtr<FSlateStyleSet> FVitruvioStyle::StyleSet = nullptr;
+TSharedPtr<ISlateStyle> FVitruvioStyle::Get()
+{
+	return StyleSet;
+}
+
+FName FVitruvioStyle::GetStyleSetName()
+{
+	static FName VitruvioStyleName(TEXT("VitruvioStyle"));
+	return VitruvioStyleName;
+}
+
+BEGIN_SLATE_FUNCTION_BUILD_OPTIMIZATION
+
+void FVitruvioStyle::Initialize()
+{
+	const FVector2D Icon64x64(64, 64);
+	const FVector2D Icon16x16(16, 16);
+	
+	if (StyleSet.IsValid())
+	{
+		return;
+	}
+
+	StyleSet = MakeShareable(new FSlateStyleSet(GetStyleSetName()));
+
+	StyleSet->SetContentRoot(IPluginManager::Get().FindPlugin(TEXT("Vitruvio"))->GetBaseDir());
+	StyleSet->SetCoreContentRoot(FPaths::EngineContentDir() / TEXT("Slate"));
+
+	FSlateStyleSet* Style = StyleSet.Get();
+	StyleSet->Set("ClassIcon.VitruvioActor", new IMAGE_BRUSH_SVG("Resources/Vitruvio", Icon16x16));
```

Comment:
We use the same svg for all icons -> why not use a local variable instead of adding a Macro?

I.e.
``` c++
FString iconPath = Style->RootToContentDir(RelativePath, TEXT(".svg"));
StyleSet->Set("ClassThumbnail.VitruvioActor", new FSlateVectorImageBrush(iconPath , Icon64x64));
```

Reply:
Good point. No need for a macro.